api/apps/v1alpha1/nimservice_types.go

```diff
@@ -1817,6 +1834,31 @@ func (n *NIMService) GetMultiNodePipelineParallelism() uint32 {
 	return 0
```

Comment:
No part of your change, but can you please update it?
`GetMultiNodePipelineParallelism()` is looking for `n.Spec.MultiNode.Parallelism.Tensor`nil check instead of `n.Spec.MultiNode.Parallelism.Pipeline`